src/components/Game.js

```diff
@@ -0,0 +1,251 @@
+import { h } from 'preact'
+import { Entity } from 'aframe-react'
+import { Presenter } from 'microcosm-preact'
+import {
+  flick,
+  ready,
+  reset,
+  startTimer,
+  stopTimer,
+  toggleSettings
+} from '../actions/game'
+import config from '../config.js'
+
+import Assets from './Assets'
+import MainScene from './MainScene'
+import Moves from './Moves'
+import Particles from './Particles'
+import Sky from './Sky'
+import Timer from './Timer'
+
+export default class Game extends Presenter {
+  getModel() {
+    return {
+      games: state => state.game.games,
+      hasWon: state => state.game.hasWon,
+      lights: state => state.game.lights,
+      ready: state => state.game.ready,
+      moves: state => state.game.moves,
+      textures: state => state.game.textures,
+      timeElapsed: state => state.game.timeElapsed
+    }
```

Comment:
This is idiomatic Microcosm, but I've been thinking a lot about these model bindings:
https://github.com/vigetlabs/microcosm/issues/378